website/src/app/app.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { User } from './user';
+import {UserServiceService} from './user-service.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+  providers: [UserServiceService]
+})
+export class AppComponent implements OnInit {
```

Comment:
I agree that this app is basically a user list now, but having user list as a separate component would be better for the future.